drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -331,6 +335,54 @@ function DragAndDropTemplates(configuration) {
         );
     };
 
+    var getFixedSizingFeedbackDetailTemplate = function(ctx) {
```

Comment:
I think this should be called `getAssessmentModeFeedbackDetailTemplate` since this has more to do with assessment mode and less to do with fixed sizing.

Reply:
@xitij2000 this will be called for fixed item sizing in assessment mode only.